thera/src/thera/reporting/Utils.scala

```diff
@@ -0,0 +1,47 @@
+package thera.reporting
+
+import fastparse.Parsed.Failure
+
+import scala.io.Source
+import scala.util.Using
+
+object Utils {
+
+  private[thera] def getLineNb(s: String, filename: String): Int = getLine(s, filename)._2
```

Comment:
Use public instead

Reply:
Dotty dropped private[this], not what we use

Reply:
Oh, ok, so we can leave this here.